src/workspace/src/hydroball_config_websocket/tests/test_hydroball_config_websocket.cpp

```diff
@@ -1,38 +1,112 @@
+/*
```

Comment:
Why is the test licensed to Peter Thorson?

Reply:
80% - 90%  comes from here https://github.com/zaphoyd/websocketpp/blob/master/tutorials/utility_client/step6.cpp